quickstart/101-managed-lustre-create-filesystem/main.tf

```diff
@@ -0,0 +1,37 @@
+resource "azurerm_resource_group" "example" { 
```

Comment:
Please define the RG as in https://github.com/Azure/terraform/blob/master/quickstart/101-resource-group/main.tf